p/themes/base-theme/template.css

```
@@ -572,6 +572,10 @@ a.btn {
 	width: 100px;
 }
 
+.header > .item.search input {
+	-webkit-appearance: none; /* csslint allow: -webkit-appearance */
```

```suggestion
.header > .item.search input {
	/* stylelint-disable property-no-vendor-prefix */
	-webkit-appearance: none;
	/* stylelint-enable property-no-vendor-prefix */
}
```

There might be a better syntax, but this one seems to work